.python-version

```diff
@@ -1 +0,0 @@
-2.7.15
```

Comment:
what was this used by? do we need to update it?

Reply:
Oops, meant to not touch this. I'll revert.
Greg added it recently in fad0587c9b. It's not obvious to me that we need it.